internal/scraper/scraper.go

```diff
@@ -824,26 +828,50 @@ func httpSettingsToBBEModule(ctx context.Context, logger zerolog.Logger, setting
 	m.HTTP.ValidHTTPVersions = make([]string, len(settings.ValidHTTPVersions))
 	copy(m.HTTP.ValidHTTPVersions, settings.ValidHTTPVersions)
 
-	m.HTTP.FailIfBodyMatchesRegexp = make([]string, len(settings.FailIfBodyMatchesRegexp))
-	copy(m.HTTP.FailIfBodyMatchesRegexp, settings.FailIfBodyMatchesRegexp)
+	m.HTTP.FailIfBodyMatchesRegexp = make([]bbeconfig.Regexp, 0, len(settings.FailIfBodyMatchesRegexp))
+	for _, str := range settings.FailIfBodyMatchesRegexp {
+		re, err := bbeconfig.NewRegexp(str)
+		if err != nil {
+			return m, err
+		}
+
+		m.HTTP.FailIfBodyMatchesRegexp = append(m.HTTP.FailIfBodyMatchesRegexp, re)
+	}
+
+	m.HTTP.FailIfBodyNotMatchesRegexp = make([]bbeconfig.Regexp, 0, len(settings.FailIfBodyNotMatchesRegexp))
+	for _, str := range settings.FailIfBodyNotMatchesRegexp {
+		re, err := bbeconfig.NewRegexp(str)
+		if err != nil {
+			return m, err
+		}
 
-	m.HTTP.FailIfBodyNotMatchesRegexp = make([]string, len(settings.FailIfBodyNotMatchesRegexp))
-	copy(m.HTTP.FailIfBodyNotMatchesRegexp, settings.FailIfBodyNotMatchesRegexp)
+		m.HTTP.FailIfBodyNotMatchesRegexp = append(m.HTTP.FailIfBodyNotMatchesRegexp, re)
+	}
```

Comment:
this is here because the new BBE version switched the type of `FailIfBodyNotMatchesRegexp`.